.github/workflows/api-compatibility.yml

```diff
@@ -0,0 +1,127 @@
+name: API Compatibility (Valkey.Glide vs StackExchange.Redis)
+
+on:
+    pull_request:
+        branches: [main]
+    workflow_dispatch:
+        inputs:
+            ser_version:
+                description: StackExchange.Redis version to compare against
+                required: false
+                default: "2.8.58"
+
+permissions:
+    contents: read
+
+jobs:
+    api-compatibility:
+        runs-on: windows-latest
+        env:
+            # Default for PRs; workflow_dispatch can override via input
+            SER_VERSION: ${{ github.event_name == 'workflow_dispatch' && inputs.ser_version || '2.8.58' }}
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  submodules: true
+
+            - name: Setup .NET
+              uses: actions/setup-dotnet@v4
+              with:
+                  dotnet-version: |
+                      8.x
+
+            # Install Rust toolchain on Windows for cargo prebuild
```

Comment:
uncomment or delete?

Reply:
Was in the middle of testing as the pipeline was failing. Will remove now that everything is working and is def not needed.